xbmc/windowing/gbm/DRMUtils.h

```diff
@@ -89,6 +90,9 @@ class CDRMUtils
   virtual bool AddProperty(struct drm_object *object, const char *name, uint64_t value) { return false; }
   virtual bool SetProperty(struct drm_object *object, const char *name, uint64_t value) { return false; }
 
+  static uint32_t ToAlpha(uint32_t fourcc);
+  static uint32_t FromAlpha(uint32_t fourcc);
+
```

Comment:
I think @pkerling mentioned it already in the linked PR and I agree with him.
Those methods need a better name, they are part of the public api and DRMUtils is not only about planes.
So the name should make it more clear that you have to feed a plane and what the result is.
Actually I think we should start to use proper method comments in the header for public methods (not needed in this PR but we should do it at some point).

Reply:
i disagree. doxy. always. not later. now.

Reply:
I'll come up with a PR to add the doxy then.